content/mentorship-for-mentees.md

```diff
@@ -29,7 +29,12 @@ It does not mean that the longer the proposal the better. It's about the quality
 In order to understand the project better and come up with reasonable solutions, it's always helpful to become familiar with Jaeger and its code base. We strongly recommend:
 
 * Going through some Jaeger tutorials, such as [this one][hotrod].
-* Trying to solve some of the [simple open issues](../get-involved/) that you can find across Jaeger repositories.
+* Review the hotrod video demo and tutorial such as [this video on youtube][hotrod_video].
+* Run the [hotrod demo yourself][hotrod_repo] as the blogs and video is outdated, it's good to get hands on.
+* Review the Jaeger architecture from the [documentation](https://www.jaegertracing.io/docs/architecture/) and understand the microservices architecture.
```

Comment:
```suggestion
* Review the [Jaeger architecture documentation](https://www.jaegertracing.io/docs/architecture/) and understand the microservices architecture.
```

> understand the microservices architecture.

what do you mean by that?

Reply:
> https://www.jaegertracing.io/docs/architecture/

better to use relative path

Reply:
Understanding of the components in the architecture is another way to phrase it, if you prefer.

I will edit to use relative on the push.